docs/content/concepts/assets/asset-jobs.mdx

```diff
@@ -20,43 +20,43 @@ description: Asset jobs are the main unit for materializing and monitoring Softw
 
 ---
 
-## Relevant APIs
+## Creating asset jobs
 
-| Name                                   | Description                                               |
-| -------------------------------------- | --------------------------------------------------------- |
-| <PyObject object="define_asset_job" /> | A function for defining a job from a selection of assets. |
+Asset jobs materialize a fixed set of assets each time they run. We'll use the following assets to demonstrate how to create a few asset jobs:
 
----
+```python file=/concepts/assets/non_argument_deps.py startafter=start_marker endbefore=end_marker
+from dagster import asset
 
-## Creating asset jobs
 
-Asset jobs materialize a fixed set of assets each time they run. Additionally, multiple jobs can target overlapping sets of assets:
+@asset
+def sugary_cereals() -> None:
+    execute_query("CREATE TABLE sugary_cereals AS SELECT * FROM cereals")
 
-```python file=/concepts/assets/build_job.py startafter=start_job_only_marker endbefore=end_job_only_marker
-from dagster import Definitions, asset, define_asset_job
 
+@asset(deps=[sugary_cereals])
+def shopping_list() -> None:
+    execute_query("CREATE TABLE shopping_list AS SELECT * FROM sugary_cereals")
+```
 
-@asset
-def asset1():
-    return [1, 2, 3]
+To create an asset job, use <PyObject object="define_asset_job" />. The topology of an asset-based job is based on the [assets](/concepts/assets/software-defined-assets) and their dependencies.
```

Comment:
`topology` is a complicated word here that confuses me more than it helps me understand. Maybe structure?